spark/v3.1/spark/src/main/java/com/netease/arctic/spark/ArcticSparkCatalog.java

```diff
@@ -206,15 +201,39 @@ public Table createTable(
     }
   }
 
-  private Schema checkAndConvertSchema(Schema schema, Map<String, String> properties) {
+  private Schema checkAndConvertSchema(StructType schema, Map<String, String> properties) {
+    Schema convertSchema;
+    boolean defaultTimeZoneUsed;
+    try {
+      CatalogMeta catalogMeta = client.getCatalog(catalog.name());
+      String catalogType = catalogMeta.getCatalogType();
+      SparkSession sparkSession = SparkSession.active();
+      if (CATALOG_TYPE_HIVE.equals(catalogType)) {
```

Comment:
Can we determine the catalog type by the `ArcticCatalog` implementation? So we may need not the client and rpc to AMS.

Reply:
Does it mean adding a new implementation in ArcticCatalog to determine the catalog type?

Reply:
Currently, there is no implementation in ArcticCatalog that can determine the type of catalog

Reply:
How about adding a `CatalogUtil#isHiveCatalog` implementation in the hive module?